### ISSUE TITLE
Is there any plan release the training scripts?

### ISSUE THREAD

**Issue by jinfagang** [ternaus/TernausNetV2#23]
Just wonder is there any plan to release training scripts? we don't any details about data preparing and loss function.....
